src/frontend/http.rs

```diff
@@ -100,6 +102,25 @@ fn plan_to_etag(plan: &LogicalPlan) -> String {
     encode(hasher.finalize())
 }
 
+// Construct a content-type header value that also includes schema information.
+fn content_type_with_schema(schema: SchemaRef) -> HeaderValue {
+    let output = schema
+        .fields
+        .iter()
+        .map(|f| format!("{}={}", f.name(), f.data_type()))
+        .collect::<Vec<String>>()
+        .join("; ");
```

Comment:
One alternative would be to use [serde serialization of the schema](https://github.com/apache/arrow-rs/blob/master/arrow-schema/src/schema.rs#L122) (so someone on the other side can get a proper arrow object), though I suspect that may end up being too convoluted.